app.js

```diff
@@ -36,6 +42,18 @@ app.use(createSession(config.session_secret, config.session_timeout, config.mong
 // add graphql endpoint
 app.use("/api/graphql", graphqlRouter);
 
+// Scheduled cronjob once a day (1am) eastern time
+cronJob.schedule("1 0 1 * * *", async () => {
```

Comment:
Based on feedback from when we implemented scheduled jobs in Bento, we should make the cronJob schedule parameter an environment variable so that we can setup test environments to run more frequently without having a code change

Reply:
Updated.